client/my-sites/customize/main.jsx

```diff
@@ -299,13 +302,15 @@ var Customize = React.createClass( {
 
 		if ( iframeUrl ) {
 			debug( 'loading iframe URL', iframeUrl );
-			let iframeClassName = this.state.iframeLoaded ? 'is-iframe-loaded' : '';
+			const siteID = this.getSite().ID;
```

Comment:
Since we're creating a `siteId` prop in the `connect()` call anyway, we should rather use that.